website/templates/docs.html

```diff
@@ -535,12 +539,19 @@ <h2 id=generics>Generics</h2>
 }
 
 <comment>// Generic code is notoriously verbose. To reduce clutter, V doesn't require you 
-// to add `&lt;T&gt;` every time, since it already knows that Repo is a generic type.</comment> 
+// to add `&lt;T&gt;` every time, since it already knows that Repo is a generic type.
+// Here the function return type is an unspecified instance of Repo.</comment> 
 fn new_repo&lt;T>(db DB) Repo {
 	return Repo&lt;T>{db: db}
 }
 
-<comment>// This is a generic function. V will generate it for every type it's used with.</comment> 
+fn (db DB) query_one&lt;T>(query string, id int) T? {
+	<comment>// Return a value of type T from the database, or an error.
+	// Implementation omitted...</comment>
+}
```

Comment:
I added this because seeing the signature seems helpful to understanding this example.